### PR TITLE
Upgrade the JHipster console to v2.1

### PR DESCRIPTION
**Pull request by PierreBesson**
- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

@jdubois I have added a readiness check and depends_on that should fix the problem you had. Can you test it again when you have the time.

### CODE REVIEW

File: generators/docker-compose/index.js [hunk 1]
```diff
@@ -53,12 +53,14 @@ module.exports = DockerComposeGenerator.extend({
...
             this.DOCKER_JHIPSTER_ELASTICSEARCH = constants.DOCKER_JHIPSTER_ELASTICSEARCH;
             this.DOCKER_JHIPSTER_LOGSTASH = constants.DOCKER_JHIPSTER_LOGSTASH;
+            this.DOCKER_JHIPSTER_IMPORT_DASHBOARDS = constants.DOCKER_JHIPSTER_IMPORT_DASHBOARDS;
             this.DOCKER_JHIPSTER_ZIPKIN = constants.DOCKER_JHIPSTER_ZIPKIN;
             this.DOCKER_CONSUL = constants.DOCKER_CONSUL;
             this.DOCKER_CONSUL_CONFIG_LOADER = constants.DOCKER_CONSUL_CONFIG_LOADER;
             this.DOCKER_PROMETHEUS = constants.DOCKER_PROMETHEUS;
             this.DOCKER_PROMETHEUS_ALERTMANAGER = constants.DOCKER_PROMETHEUS_ALERTMANAGER;
             this.DOCKER_GRAFANA = constants.DOCKER_GRAFANA;
+            this.DOCKER_COMPOSE_VERSION = constants.DOCKER_COMPOSE_VERSION;
```
Comment (deepu105) [inline generators/docker-compose/index.js]: The constant name is bit misleading. I thought it refers to the `docker-compose` executable version

File: generators/generator-constants.js [hunk 2]
```diff
@@ -31,17 +31,20 @@ const DOCKER_ELASTICSEARCH = 'elasticsearch:2.4.1';
...
+const DOCKER_JHIPSTER_IMPORT_DASHBOARDS = 'jhipster/jhipster-import-dashboards:v2.1.0';
+const DOCKER_JHIPSTER_ZIPKIN = 'jhipster/jhipster-zipkin:v2.1.0';
 const DOCKER_CONSUL = 'consul:0.8.3';
 const DOCKER_CONSUL_CONFIG_LOADER = 'jhipster/consul-config-loader:v0.2.2';
 const DOCKER_PROMETHEUS = 'prom/prometheus:v1.6.3';
 const DOCKER_PROMETHEUS_ALERTMANAGER = 'prom/alertmanager:v0.6.2';
 const DOCKER_GRAFANA = 'grafana/grafana:4.3.2';
 const DOCKER_JENKINS = 'jenkins:latest';
 
+const DOCKER_COMPOSE_VERSION = '2.1';
```
Comment (deepu105) [inline generators/generator-constants.js]: Shouldnt this be called, `DOCKER_COMPOSE_RELEASE_VERSION` or something clearer

File: generators/docker-compose/index.js (hunk 1 above)
Comment (PierreBesson) [inline generators/docker-compose/index.js]: You are right, it refers to the docker-compose file format version. I will rename this to DOCKER_COMPOSE_FORMAT_VERSION

**Comment by jdubois** [comment]
@PierreBesson can you solve the conflicts?
And this time, does it work "out of the box"?

**Comment by PierreBesson** [comment]
I'm closing this as I want to do the upgrade using another approach. I don't think it's a good idea to use the docker compose healthcheck feature because its not compatible with swarm.

**Pull request closed without merging**